Fix segfault in chan_iax2.
Upstream issue #11602, fixed in upstream r93949.

 -- Faidon Liambotis <paravoid@debian.org>

--- a/channels/chan_iax2.c
+++ b/channels/chan_iax2.c
@@ -2696,7 +2696,7 @@
 			}
 		}
 	}
-	if (!var) { /* Last ditch effort */
+	if (!var && peername) { /* Last ditch effort */
 		var = ast_load_realtime("iaxpeers", "name", peername, NULL);
 		/*!\note
 		 * If this one loaded something, then we need to ensure that the host
